bot/constants.py

```diff
@@ -270,7 +271,7 @@ class Roles(NamedTuple):
     moderator = 267629731250176001
     owner = 267627879762755584
     helpers = int(environ.get("ROLE_HELPERS", 267630620367257601))
-    core_developers = 587606783669829632
+    core_developers = 814190307319087172
```

Comment:
Did you forget to change this back to the correct ID?

Reply:
Whoops yeah